cmd/metricscollector/v1beta1/tfevent-metricscollector/requirements.txt

```diff
@@ -1,4 +1,4 @@
-psutil==5.8.0
+psutil==5.9.4
```

Comment:
To resolve the following error, we must upgrade the psutil version.

ref: https://github.com/giampaolo/psutil/issues/2165